docs/csharp/programming-guide/statements-expressions-operators/default-value-expressions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "default value expressions (C# Programming Guide)"
+
+ms.date: "2017-08-03"
+ms.prod: .net
+
+
+ms.technology: 
+  - "devlang-csharp"
+
+ms.topic: "article"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "generics [C#], default keyword"
+  - "default keyword [C#], generic programming"
+ms.assetid: b9daf449-4e64-496e-8592-6ed2c8875a98
+caps.latest.revision: 22
+author: "BillWagner"
+ms.author: "wiwagn"
+
+translation.priority.ht: 
```

Comment:
I'm not sure that these are needed -- at least, when I've included them in new topics, Maira told me to remove them.